airbyte-integrations/connectors/source-microsoft-lists/metadata.yaml

```diff
@@ -26,6 +26,14 @@ data:
   releaseDate: 2024-10-18
   releaseStage: alpha
   supportLevel: community
+  connectorTestSuitesOptions:
+    - suite: unitTests
+      testSecrets:
+        - name: SECRET_SOURCE-MICROSOFT-LISTS_CREDS
+          fileName: config.json
+          secretStore:
+            type: GSM
+            alias: airbyte-connector-testing-secret-store
```

Comment:
Shared in DM but posting here for posterity...

Apparently unitTests don't support secrets. And manifest-only connectors don't support integration tests...

Suggestion is to move these into integration tests and add that capability (integration tests for manifest-only connectors) into airbyte-ci in a future iteration.